src/pages/Product/Product.tsx

```diff
@@ -36,14 +37,12 @@ export default function Product(): ReactElement {
 
   return (
     <div className={styles.product} data-testid="product">
-      {status === 'loading' ? (
-        <Loader type="spinner" />
-      ) : error ? (
-        <NotFound />
-      ) : (
+      {status === 'loading' && <Loader type="spinner" />}
```

Comment:
We have discussed that the type of Loader will be 'text'

Reply:
Yes, forgot about this Loader.